health/state.go

```diff
@@ -98,11 +101,33 @@ func (t *Tracker) CurrentState() *State {
 		wm[w.Code] = *w.unhealthyState(ws)
 	}
 
+	for id, msg := range t.lastNotifiedControlMessages {
+		code := WarnableCode(id)
+		wm[code] = UnhealthyState{
+			WarnableCode:        code,
+			Severity:            severityFromTailcfg(msg.Severity),
+			Title:               msg.Title,
+			Text:                msg.Text,
+			ImpactsConnectivity: msg.ImpactsConnectivity,
```

Comment:
Do we need a `DependsOn: []WarnableCode("wantrunning-false"),` or similar to suppress these control health messages when our connection has dropped temporarily? Is that what would happen? Is that when `DependsOn` does? 😅

Reply:
I would not change this now (since existing warnable does not specify DependsOn), but it's worth leaving a TODO to re-evaluate. In addition to `wantrunning-false` (meaning the client is not connected), we might want to hide these on `not-in-map-poll` and `mapresponse-timeout` perhaps.

Reply:
Noted as TODO.